DemoqaProject/pageObjects/Elements.cs

```diff
@@ -7,10 +7,323 @@ namespace DemoqaProject.PageObjects
+        {
+            textBoxButton.Click();
+            fullNameInput.SendKeys(fullname);
+            emailInput.SendKeys(email);
+            currentAddressInput.SendKeys(currentAddress);
+            permanentAddressInput.SendKeys(permanentAddress);
+            submitButton.Click();
+            
+            IJavaScriptExecutor js=driver as IJavaScriptExecutor;
+            js.ExecuteScript("userForm.reset()");
+            
+        }
+
+        //Getting text of elements
+        public string FullNameText()
+        {
+            return fullnameText.Text;
+        }
+        
+        public string EmailText()
+        {
+            return emailText.Text;
+        }
+        
+        public string CurrentAdText()
+        {
+            return currentAddressText.Text;
+        }
+        
+        public string PerAdText()
+        {
+            return permanentAddressText.Text;
+        }
+
+        //Getting placeholders
+        public string GetFNamePlaceholder()
+        {
+            return fullNameInput.GetAttribute("placeholder");
+        }
+
+        public string GetEmailPlaceholder()
+        {
+            return emailInput.GetAttribute("placeholder");
+        }
+
+        public string GetCurrentAddPlaceholder()
+        {
+            return currentAddressInput.GetAttribute("placeholder");
+        }
+
+        public void NavigateToCheckBox()
+        {
+            checkBoxButton.Click();
+            selectAll.Click();
+        }
+
+        public void Count()
+        {
+            ReadOnlyCollection<IWebElement> checkbox = driver.FindElements(By.XPath("//span[@class='text-success']"));
+            Console.WriteLine(checkbox.Count);
+            if (checkbox.Count==17)
+            {
+                Console.WriteLine("Selected all");
+            }
+            else if (!selectAll.Selected)
+            {
+                Console.WriteLine("Deselected all");
+            }
+            selectAll.Click();
+        }
+
+        public void SpecificCategories()
+        {
+            //Select specific category and verify
+            arrowButton.Click();
+            if (!desktopButton.Selected)
+            {
+                desktopButton.Click();
+                Console.WriteLine("Category Desktop selected");
+            }
+            else
+            {
+                Console.WriteLine("Category Desktop not selected");
+            }
+        }
+
+        public void SelectGroupAndUncheckSub()
+        {
+            ReadOnlyCollection<IWebElement> checkbox = driver.FindElements(By.XPath("//button[@class='rct-collapse rct-collapse-btn']"));
+            checkbox[1].Click();
+            if (!commandsButton.Selected)
+            {
+                commandsButton.Click();
+                Console.WriteLine("Sub group deselected");
+            }
+        }
+
+        public void ExpendAndCollapseAll()
```

Comment:
What is purpose of this method? Why we expand and collapse it?

Reply:
Just checking expand and collapse buttons